### DISCUSSION TITLE
Is there currently a way to export your plays?

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#16279] [Q&A] [answered]
On stable I remember downloading your plays would download the osu file but on lazer it all seems to be ingame only.

**Comment by bdach** [comment, marked as answer]
yes, right click on score and select "export"

![2021-12-29-183608_774x173_scrot](https://user-images.githubusercontent.com/20418176/147688805-1eeaa872-5b20-4c4b-a78c-ef469fbc8457.png)

**Reply by localcake** [reply]
Thank you, I have no idea how I only found that out now after so many hours of using it.
